src/compactor.rs

```diff
@@ -483,11 +497,19 @@ mod tests {
 
     #[tokio::test]
     async fn test_compactor_compacts_l0() {
-        // given:
```

Comment:
Nit: for consistency, probably want to keep this.

Reply:
Updated